src/components/Patient/PatientRegistration.tsx

```diff
@@ -0,0 +1,1076 @@
+import careConfig from "@careConfig";
+import { useQuery } from "@tanstack/react-query";
+import { navigate } from "raviger";
+import { Fragment, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+import CareIcon from "@/CAREUI/icons/CareIcon";
+import SectionNavigator from "@/CAREUI/misc/SectionNavigator";
+
+import { Button } from "@/components/ui/button";
+import { Checkbox } from "@/components/ui/checkbox";
+import { InputErrors } from "@/components/ui/errors";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { RadioGroup, RadioGroupItem } from "@/components/ui/radio-group";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import { Textarea } from "@/components/ui/textarea";
+
+import DialogModal from "@/components/Common/Dialog";
+import Loading from "@/components/Common/Loading";
+import Page from "@/components/Common/Page";
+import DuplicatePatientDialog from "@/components/Facility/DuplicatePatientDialog";
+import TransferPatientDialog from "@/components/Facility/TransferPatientDialog";
+import { PatientModel } from "@/components/Patient/models";
+
+import useAppHistory from "@/hooks/useAppHistory";
+
+import {
+  BLOOD_GROUPS,
+  DOMESTIC_HEALTHCARE_SUPPORT_CHOICES,
+  GENDER_TYPES,
+  OCCUPATION_TYPES,
+  RATION_CARD_CATEGORY,
+  SOCIOECONOMIC_STATUS_CHOICES,
+} from "@/common/constants";
+import countryList from "@/common/static/countries.json";
+import { validatePincode } from "@/common/validation";
+
+import * as Notification from "@/Utils/Notifications";
+import routes from "@/Utils/request/api";
+import query from "@/Utils/request/query";
+import useMutation from "@/Utils/request/useMutation";
```

Comment:
use useMutation from tanstack query instead. this is deprecated...

https://docs.ohc.network/docs/care/development/data-fetching-in-care/